hedera-node/src/main/java/com/hedera/services/store/contracts/HederaStackedWorldStateUpdater.java

```diff
@@ -144,7 +146,7 @@ public WorldUpdater updater() {
 		return new HederaStackedWorldStateUpdater(
 				(AbstractLedgerWorldUpdater) this,
 				worldState,
-				trackingLedgers().wrapped());
+				trackingLedgers().wrapped(sideEffectsTracker), sideEffectsTracker);
```

Comment:
It shouldn't be necessary to propagate a `SideEffectsTracker` here (consider that only the EVM/Besu machinery calls this method, and it doesn't know anything about HTS precompiles or `TransactionLedger`s.)